frontend/packages/console-shared/src/components/status/StatusIconAndText.tsx

```diff
@@ -16,14 +19,15 @@ const StatusIconAndText: React.FC<StatusIconAndTextProps> = ({
   iconOnly,
   noTooltip,
   className,
+  description,
```

Comment:
I think it would be better to use `<SecondaryStatus>` to add the description to the status. It is a concept which is already used and available (see NodeStatus.tsx for example). Your change is introducing another implementation of the same concept which I think we should avoid.

Is your reasoning for doing this to have the icon vertically aligned evenly among the status and description? I'd rather avoid this as it is not used anywhere else. Current usage in console is:
```
 ----------------
| <icon><status> |
| <substatus>    |
 ----------------
```

Reply:
Thats exactly the reason. Overview pages also use vertically aligned icon and we have the same in UXD mockups. So at some point we should start aligning to just one style.